reference/format/datagrid.rst

```diff
@@ -0,0 +1,166 @@
+
+``columns`` (**type**: ``map``)
+
+    Each key of the map corresponds to a property of a select entry that should be displayed in the
+    grid. For each column you have to pass a map of options to configure how the column is shown:
+
+    ``choices`` (**type**: ``map``)
+
+        When using the ``select`` type, this options provides a mapping of stored values to their
+        human-readable representation.
+
+        When the ``type`` option is set to ``translatable``, this is treated as an expression that
+        will be evaluated to gain the value's translated representation.
+
+    ``frontend_type`` (**type**: ``string``)
+
+        The widget type that renders the value. Available types that are shipped with OroPlatform
+        by default are ``currency``, ``date``, ``datetime``, ``html``, ``integer``, ``select`` and ``string``.
+
+    ``label`` (**type**: ``string``)
+
+        The column headline which is a string that will be passed to the translator before being
+        displayed.
+
+    ``label`` (**type**: ``string``)
+
+        This option is only available when the ``frontend_type`` is ``html``. Currently, only
+        ``twig`` is supported as a value to indicate that a Twig template should be rendered.
+
+    ``template`` (**type**: ``string``)
+
+        This option is only available when the ``frontend_type`` is ``html`` and the value of the
+        ``type`` option is ``twig``. Its value is a template reference.
+
+    ``type`` (**type**: ``string``)
+
+        When the ``frontend_type`` is ``string``, this option can be set to ``translatable`` to
+        provide translated values.
+
+        When it is set to ``twig``, the Twig template referenced to with the ``template`` option
+        will be rendered.
+
+``extends`` (**type**: ``string``)
+
+    You can reuse existing datagrid configurations by passing their name here.
+
+``filters`` (**type**: ``map``)
+
+    This option currently only provides the ``columns`` key which maps column names to a map which
+    configures how a particular column can be filtered. Available options to configure how to
+    filter a column are:
+
+    ``data_name`` (**type**: ``string``)
+
+        The name of the column from the data source whose values will be filtered.
+
+    ``type`` (**type**: ``string``)
+
+        The type of the filter to be used in the UI.
+
+``options`` (**type**: ``map``)
+
+    Available additional options are:
+
+    ``base_datagrid_class`` (**type**: ``string`` **default**: ``Oro\Bundle\DatagridBundle\Datagrid\Datagrid``)
+
+        With this option, you can switch the datagrid class to a custom implementation.
+
+    ``export`` (**type**: ``boolean`` **default**: ``false``)
+
+        When enabled, the user can export the datagrid in CSV format.
+
+``properties`` (**type**: ``map``)
+
+    This is used for two things: configure how to determine the id of each row and configure links
```

Comment:
To be more precise, properties are used to pass some additional data that is not defined in columns lists - these are entity IDs, action URLs etc.